src/main/java/frc/robot/commands/CmdShooterBedSetBedAngle.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.shooter.ShooterBed;
+
+public class CmdShooterBedSetBedAngle extends Command
+{
+    private ShooterBed _shooterBed;
+    private Rotation2d _bedAngle;
+
+    public CmdShooterBedSetBedAngle(ShooterBed shooterBed, Rotation2d bedAngle)
+    {
+        _shooterBed = shooterBed;
+        _bedAngle   = bedAngle;
+        addRequirements(_shooterBed);
+    }
+
+    @Override
+    public void initialize()
+    {
+        _shooterBed.setAngle(_bedAngle);
+    }
+
+    @Override
+    public boolean isFinished()
+    {
+        return true;
```

Comment:
We should add a method to the subsystem that tells us whether or not the shooter bed is at the PID setpoint. Then, we should modify this function to only terminate the command once we're at the setpoint.

This may be crucial when we start including this command in command groups, like for autonomous